layouts/hub-page.tsx

```diff
@@ -29,7 +29,11 @@ const HubPageLayout = ({ children }: { children: React.ReactNode }) => {
 
   const { toolList, selectedTool, selectedFilter, userOrg } = useNav(repositories);
 
-  let isOwner = !!(userId && insight && `${userId}` === `${insight.user.id}`);
+  let canEdit = false;
+  const membership = insight?.members.find((member) => member.user_id === Number(userId));
+  if (membership !== undefined && (membership.access === "edit" || membership.access === "admin")) {
```

Comment:
refactor (blocking): same as above

Reply:
I don't think we need to promote this to a function, but if we start doing this check a lot, it may make sense.